### DISCUSSION TITLE
Netlify? New hosting website possible

### DISCUSSION THREAD

**Discussion by Anon10104-2** [QuiteAFancyEmerald/InvisiProxy#323] [Ideas]
I was looking for heroku alternatives and found netlify they allow you to upload gits i tried with this and got this "It looks like this is a Jekyll site, but you don't have a Gemfile. This will most likely lead to a broken build. Check out our [Migrating your Jekyll site to Netlify](https://www.netlify.com/blog/2017/05/11/migrating-your-jekyll-site-to-netlify/) guide to fix this issue before deploying your site." i was wondering if its still possible to launch

**Comment by BigFatJoeMama** [comment]
Only hosts static sites
